consensus/core/src/dag_state.rs

```diff
@@ -118,7 +118,7 @@ impl DagState {
         let cached_rounds = context.parameters.dag_state_cached_rounds as Round;
         let num_authorities = context.committee.size();
 
-        let genesis = genesis_blocks(context.clone())
+        let genesis = genesis_blocks(context.as_ref())
```

Comment:
consistency:
in the file `core.rs` you used `&context` and here you use `context.as_ref()`.